src/main/java/org/opentripplanner/util/OTPFeature.java

```diff
@@ -27,6 +27,7 @@
     SandboxAPILegacyGraphQLApi(false),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
The code line is a call to a constructor. OTP have lot of code that is "out-of-date" with the current OTP CodeStyle. New code should follow the code-style, but we leave the old code for now.